modules/python/Makefile

```diff
@@ -1,28 +1,30 @@
 PYTHON := $(shell which python 2>/dev/null)
 POETRY := $(shell which poetry 2>/dev/null)
 
-PACKAGE ?= src
+PY_FILES ?= $(shell git ls-files "*.py")
+PYTEST_FILES ?= tests
```

Comment:
I think using "PACKAGE" here is a bit limiting:
* as used with `mypy` it only works on a single directory
* doesn't make sense as a name if we want to check multiple directories (e.g. `src` and `tests`)

Instead, I added `PY_FILES` for running all linting/formatting/typing checks and `PYTEST_FILES` for directories containing tests.